README.md

```diff
@@ -42,3 +42,10 @@ For example, if the instance connection name is `foo:bar:baz`, the JDBC URL
 would be 
 `jdbc:mysql://google/mydb?cloudSqlInstance=foo:bar:baz&socketFactory=com.google.cloud.sql.mysql.SocketFactory`
 
+## Credentials
```

Comment:
Good point, I've opened #22 to add k8s information. I'll need some extra time to do that as I want to get it working myself first.